recursive_art.py

```diff
@@ -3,10 +3,10 @@
 
 import random
```

Comment:
Looks really good - great use of lambda functions here. There's not much else to say.